classes/Tools14.php

```diff
@@ -2410,7 +2410,7 @@ public static function copy($source, $destination, $stream_context = null)
         }
 
         if (self::shouldUseFopen($source)) {
-            $sourceFile = fopen($source);
+            $sourceFile = fopen($source, 'r');
```

Comment:
```suggestion
            $sourceFile = fopen($source, 'rb');
```

The default behavior was in bytes, the destination file is `wb`, seems to be logic to have `rb` instead of `r`

Reply:
The doc talks about Notepad issues for the reasons to add 'b' mode https://www.php.net/manual/fr/function.fopen.php

But why not :D

Done ;)